Editor/Resources/EditorWindow/Components/ConnectToFleetInput.uxml

```diff
@@ -0,0 +1,40 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="./common.uss"/>
+    <ui:VisualElement name="AnywherePageCreateFleet" class="separator">
+        <ui:VisualElement name="AnywherePageFleetName" class="form-row">
+            <ui:Label name="AnywherePageCreateFleetName" class="form-row__label" text="Fleet Name"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:TextField name="AnywherePageCreateFleetInput" class="text-input" value="GameName-AnywhereFleet"/>
+                <ui:Label name="AnywherePageCreateFleetHint" class="input-hint" text="Fleet Name must have 1–1024 characters. Valid characters are A-Z, a-z, 0-9, _ and - (hyphen)"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement class="form-row">
+            <ui:VisualElement class="form-row__input form-row__input--no-label separator separator--horizontal">
+                <ui:Button name="AnywherePageCreateFleetButton" class="button button--primary" text="Create Anywhere Fleet"/>
+                <ui:Button name="AnywherePageCreateFleetCancelButton" class="button" text="Cancel"/>
+            </ui:VisualElement>
+        </ui:VisualElement>
+    </ui:VisualElement>
+    <ui:VisualElement name="AnywherePageConnectFleet" class="separator">
+        <ui:VisualElement name="AnywhereFleetName" class="form-row">
+            <ui:Label name="AnywherePageConnectFleetLabel" class="form-row__label" text="Fleet Name"/>
```

Comment:
Same as above comment, let's use a fixed naming convention. So in this case `AnywherePageConnectFleet<ComponentType>` where ComponentType is `Name`, `Title`, `Label` etc

Reply:
Updated